CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@ Install the recommended [plugins](./.vscode/extensions.json) for linter warnings
 
 `yarn test`
 
-_Before running this command, make sure Docker is running. See [Testing](#Testing) section below for more details._
+_Before running this command, make sure Docker is running. See [Testing](#unit-testing-react) section below for more details._
```

Comment:
it's about Docker so it should point to visual testing, not unit testing

Reply:
Updated to point to visual testing css. Updated text to match content better.